Formula/p/pawk.rb

```diff
@@ -0,0 +1,25 @@
+class Pawk < Formula
+  desc "Python line processor (like AWK)"
+  homepage "https://github.com/alecthomas/pawk"
+  url "https://github.com/alecthomas/pawk/archive/refs/tags/v0.8.0.tar.gz"
```

Comment:
looks like the pypi source is newer?

https://files.pythonhosted.org/packages/6c/90/2165e9fedd33ac172899aa3df6754971d720bf07eef2a0b049db15a7ad69/pawk-0.8.1.tar.gz

Reply:
Upstream has resolved this issue.